addons/modules/functions/fnc_gui_tracers.sqf

```diff
@@ -44,6 +44,13 @@ private _cfgWeapons = configFile >> "CfgWeapons";
     _ctrlWeapon lnbSetTooltip [[_index, 0], format ["%1\n%2", _name, _x]];
     _ctrlWeapon lnbSetPicture [[_index, 0], _icon];
     _ctrlWeapon lnbSetData [[_index, 0], _x];
+
+    private _mod = [_config] call EFUNC(common,getDLC);
+
+    if (_mod != "") then {
+        private _logo = modParams [_mod, ["logo"]] param [0, ""];
+        _ctrlWeapon lnbSetPicture [[_index, 2], _logo];
+    };
```

Comment:
Extract this to a function, because it is 100% repeated from line :79 to :85.

Reply:
Decided to move the entire populate list handling to a common function.